### PR TITLE
Minor edit to scikit-learn docs.

### PR DESCRIPTION
**Pull request by koaning**
I saw room for improvement.

### CODE REVIEW

File: docs/tutorial/scikit-learn.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ## Scikit-Learn
 
-Some of the languages inside of this package can be used in scikit-learn pipelines.
-The spaCy and fasttext pipelines have compatible `.fit()` and `.transform()` methods
-implemented. That means that you could write code like this:
+Many of the language-backends inside of this package can be used in scikit-learn pipelines.
+We've implemented a compatible `.fit()` and `.transform()` which means that
```
Comment (mkaze) [inline docs/tutorial/scikit-learn.md:4]: Shouldn't an "API" be after the `transform()`?!

File: docs/tutorial/scikit-learn.md [hunk 2]
```diff
@@ -74,4 +74,6 @@ There's a few caveats to be aware of though. Fasttext as well as spaCy cannot be
 so that means that you won't be able to save a pipeline if there's a whatlies component
 in it. This also means that you cannot use a gridsearch. Where possible we try to
 test against scikit-learn's testing utilities but for now the usecases should assume that you
-cannot use `GridSearchCV` and that you cannot pickle to disk.
\ No newline at end of file
+cannot use `GridSearchCV` and that you cannot pickle to disk. 
+
+If you see a way to properly support this in general; let us know on github by creating an [issue](https://github.com/RasaHQ/whatlies/issues).
```
Comment (mkaze) [inline docs/tutorial/scikit-learn.md:79]: I think comma should be used here instead of semi-colon.

**Review by mkaze — APPROVED** [review]
Looks good to me. Thanks!

**Comment by koaning** [comment]
I see a bug but that can't be related to this documentation change. It seems related to IVIS and it is now also appearing in other pull requests. I'll merge this and adress this issue here: https://github.com/RasaHQ/whatlies/pull/226.

**Pull request merged**